cloudinit/config/schema.py

```diff
@@ -449,7 +468,11 @@ def handle_schema_args(name, args):
         except RuntimeError as e:
             error(str(e))
         else:
-            print("Valid cloud-config file {0}".format(args.config_file))
+            if args.config_file is None:
+                cfg_name = "system userdata"
+            else:
+                cfg_name = args.config_file
+            print("Valid cloud-config: {0}".format(cfg_name))
```

Comment:
Obviously a nit, but `print` takes `*args` and prints them:

```suggestion
            print("Valid cloud-config:", cfg_name)
```

```python
In [1]: print("a", "b")                                                                                                                                                                                                                                                                   
a b
```